VTEX - Giftcard API.json

```diff
@@ -1253,153 +1253,159 @@
                 }
             },
             "Cart": {
-                "title": "Cart",
-                "required": [
-                    "grandTotal",
-                    "relationName",
-                    "redemptionCode",
-                    "discounts",
-                    "shipping",
-                    "taxes",
-                    "items",
-                    "itemsTotal"
-                ],
-                "type": "object",
-                "properties": {
-                    "grandTotal": {
-                        "type": "number",
-                        "default": 122
-                    },
-                    "relationName": {
-                        "type": "string",
-                        "nullable": true,
-                        "default": "insert name here"
-                    },
-                    "redemptionCode": {
-                        "type": "string",
-                        "nullable": true,
-                        "default": "example code"
-                    },
-                    "discounts": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 0
-                    },
-                    "shipping": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 0
-                    },
-                    "taxes": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 12
-                    },
-                    "items": {
-                        "type": "array",
-                        "items": {
-                            "$ref": "#/components/schemas/Item"
-                        },
-                        "description": ""
-                    },
-                    "itemsTotal": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 100
-                    }
-                },
-                "example": {
-                    "grandTotal": 123.1,
-                    "relationName": null,
-                    "redemptionCode": null,
-                    "discounts": 0,
-                    "shipping": 0,
-                    "taxes": 12,
-                    "items": [
-                        {
-                            "productId": "1",
-                            "id": "1",
-                            "refId": "12",
-                            "name": "Product Name",
-                            "price": 100,
-                            "quantity": 1
-                        }
-                    ],
-                    "itemsTotal": 100
-                }
-            },
+				"title": "Cart information.",
```

Comment:
This component is used in a request body, therefore it must abide by the OAS guidelines for request bodies, please include examples to each field.

See our [openapi template](https://github.com/vtex/openapi-schemas/blob/master/templates/VTEX%20-%20Template%20openAPI.jsonc)

Reply:
Corrigido.